tests/load_tests/test_queue_dispatcher.py

```diff
@@ -67,10 +68,13 @@ def benchmark_queue_dispatcher(queue_backend: str,
     def enqueue_requests():
         for _ in range(num_requests):
             body = payloads.StatusBody()
+            body.env_vars = dict({
+                constants.USER_ID_ENV_VAR: 'dummy',
```

Comment:
USER_ID may have to be a valid hex with 8 characters to be correctly picked up. Please check https://github.com/skypilot-org/skypilot/blob/7e9586c1d43e46332d9ef2a2f61fd33fc2cb3991/sky/utils/common_utils.py#L96

Reply:
Thanks! I think the test succeed since it does not actually touch the real execution path, will update!